dart/gui/osg/CMakeLists.txt

```diff
@@ -8,11 +8,10 @@ if(DART_BUILD_GUI_OSG)
 
   find_package(OpenSceneGraph 3.0 QUIET
     COMPONENTS osg osgViewer osgManipulator osgGA osgDB)
-  if(OPENSCENEGRAPH_FOUND)
+  if(OPENSCENEGRAPH_FOUND AND OSG_FOUND)
```

Comment:
Oh, you already mentioned about this in the post. Sorry, wasn't paying sufficient attention. :sweat: I think it would be much great if we add a simple comment why we check both of them.